SPECIFICATION.md

```diff
@@ -51,17 +51,18 @@ The current state of the API allows multiple authentication credentials. The cli
 
 - Must allow a combination (one or more) of the following credential types:
     - Must allow an API Key and Secret (current API)
-    - Must allow a signature secret (use for _some_ API requests, and to validate WebHook signatures)
-    - Must allow multiple private keys for JWT authentication (replacing key and secret)
-    - May allow OAuth1 token for authentication
-- Must not allow more than one credential per type.
+    - Should allow a signature secret (use for _some_ API requests, and to validate WebHook signatures)
+    - Must allow a private key for JWT authentication (replacing key and secret)
```

Comment:
Should this be stated in a similar way to signature secret? e.g. 

Must allow a private key for JWT authentication (required for _some_ API requests)